src/components/legend/constants.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+    scrollBarWidth: 4,
```

Comment:
How hard would it be to factor in `ry` in the legend width calculation so that e.g.

![image](https://cloud.githubusercontent.com/assets/6675409/13128816/abca27e0-d5a6-11e5-9d4f-83a12d73bdaf.png)

does not overlap long legend items?